src/types/transactions/index.ts

```diff
@@ -16,14 +16,26 @@ import {
   ApplicationClearStateTransaction as AppClearStateTxn,
   ApplicationNoOpTransaction as AppNoOpTxn,
 } from './application';
-
+import SuggestedParamsRequest from '../../client/v2/algod/suggestedParams';
 // Utilities
 export { TransactionParams, TransactionType, SuggestedParams } from './base';
 export {
   MustHaveSuggestedParams,
   MustHaveSuggestedParamsInline,
 } from './builder';
-export { EncodedTransaction, EncodedSignedTransaction } from './encoded';
+export {
+  EncodedTransaction,
+  EncodedSignedTransaction,
+  EncodedAssetParams,
+  EncodedGlobalStateSchema,
+  EncodedLocalStateSchema,
+  EncodedLogicSig,
+  EncodedMultisig,
+  EncodedMultisigBlob,
+  EncodedSubsig,
+} from './encoded';
+export { MultisigMetadata } from '../multisig';
```

Comment:
Instead of modifying this file, could you create a new file `src/types/index.ts` that exports the types from `src/types/multisig.ts`, `src/types/address.ts`, and `src/types/transactions/index.ts` (probably with `export * from './transactions'`)?

I'd like to avoid importing files from a parent directly for the sole purpose of exporting things from them.

Reply:
yes